app/helpers/generic_phrases_helper.rb

```diff
@@ -1,17 +1,95 @@
 module GenericPhrasesHelper
-  def map_to_table(generic_phrases)
-    generic_phrases.map do |generic_phrase|
+  def generic_phrase_table_headers(presenter)
```

Comment:
I think this should be in the presenter but I think it's a poor use of time to refactor it now given that we need to jointly decide where stuff lives and how it works. Pending that I think this is ok for the time being

Reply:
One to discuss later on. For now I agree.